src/lightning_app/cli/commands/connection.py

```diff
@@ -19,6 +19,7 @@
 _PPID = str(psutil.Process(os.getpid()).ppid())
 _LIGHTNING_CONNECTION = os.path.join(_HOME, ".lightning", "lightning_connection")
 _LIGHTNING_CONNECTION_FOLDER = os.path.join(_LIGHTNING_CONNECTION, _PPID)
+_CLEAN_LIGHTNING_CONNECTION = True
```

Comment:
I couldn't see from the code why a global variable is required. Could you explain it and/or write a small comment in the source code? Is there no way to avoid it?

Reply:
If needed, it could be moved to constants as an env var and maybe renaming it to `_ENABLE_CLEAN_LIGHTNING_CONNECTION`

Reply:
So this is there exclusively for simplifying testing. Admittedly not ideal.
Outside of testing (or debugging) we need this constant to be True.

Reply:
I reworked monkeypatching, this has been removed